content/python/concepts/built-in-functions/terms/iter/iter.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'iter()'
+Description: 'Returns an iterator object for a given object'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Built-in Functions'
+  - 'Iterators'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`iter()`** function is a built-in Python function used to create an **iterator** object.
+
+## Syntax
+
+The `iter()` function has two main forms:
+
+```pseudo
+iter(object[, sentinel])
+```
+
+- `object`: Required. An object. The `iter()` function returns an iterator object for this.
+- `sentinel`: Optional. If set, the iterator will call the object with no arguments for each call to its `__next__()` method. If the value returned is equal to `sentinel`, `StopIteration` will be raised, otherwise the value will be returned.
+
+## Example
+
+Here is an example of how you might use the `iter()` function:
+
+```py
+# create a list
+fruits = ["apple", "banana", "cherry"]
+
+# get an iterator object
+fruit_iter = iter(fruits)
+
+# iterate over the list using the iterator
+print(next(fruit_iter))
+print(next(fruit_iter))
+print(next(fruit_iter))
+```
+
+In the above code snippet, we first create a list of fruits. We then use the `iter()` function to get an iterator object for the list. Finally, we use the `next()` function to iterate over the items in the list.
```

Comment:
```suggestion
In the code above a list is declared, and the `iter()` function is used to get an iterator object for the list. Finally, the `next()` function is used to step through the items in the list.
```

Reply:
Generally, we try to avoid the use of first and second person pronouns in docs (I, you, we)